third_party/2/google/protobuf/internal/containers.pyi

```diff
@@ -0,0 +1,36 @@
+from google.protobuf.message import Message
+from typing import Sequence, TypeVar, Generic, Any, Iterator, Iterable, Union
+
+_T = TypeVar('_T')
+class BaseContainer(Generic[_T], Sequence[_T]):
```

Comment:
You don't need to inherit specifically from Generic (but it's harmless, so feel free to leave this unchanged).

More importantly though, this class doesn't actually expose the entire Sequence interface; for example, there is no `.count()` method. Why don't you simply provide stubs for the methods that actually exist in the implementation?